elide-graphql/src/main/java/com/yahoo/elide/graphql/QueryRunner.java

```diff
@@ -194,8 +196,9 @@ private ElideResponse executeGraphQLRequest(ObjectMapper mapper, User principal,
             if (jsonDocument.has(OPERATION_NAME) && !jsonDocument.get(OPERATION_NAME).isNull()) {
                 executionInput.operationName(jsonDocument.get(OPERATION_NAME).asText());
             }
-
             executionInput.variables(variables);
+            queryRunnerAcceptQuery(requestScope, principal, query);
```

Comment:
Same comment I had earlier - this accept should move into the try statement itself (which means we need an interface change there).